governance/tsc/tsc-2025-10-27.md

```diff
@@ -0,0 +1,150 @@
+# Servo TSC Meeting October 2025
+
+* Date: Monday 27th Oct 2025 at 10:00 UTC
+* Location: https://meet.jit.si/Servo-TSC
+* Agenda: https://github.com/servo/project/issues/185
+
+## Agenda
+
+* AOB
+
+## Notes
+
+Attending:
+* TSC members: atbrakhi, emilio, delan, jschwe, mrego, mrobinson, nicoburns
```

Comment:
Not sure who was on the call, as I couldn't only join on listening mode part of it. Please review if we're missing someone on this list. Thanks!